<?xml version="1.0"?>
<launch>
	<node name="gcar_planner" pkg="gcop_ctrl" type="gcar_ddp_planner" output="screen" />

</launch>
